UnityProject/Assets/Scripts/Core/SecureStuff/AllowedReflection.cs

```diff
@@ -146,15 +276,16 @@ public static T CreateInstance<T>() where T : IAllowedReflection, new()
 		}
 
 
-		public static Dictionary<Type,List<MethodsAndAttributee<T>>> GetFunctionsWithAttribute<T>() where T : BaseAttribute
+		public static Dictionary<Type, List<MethodsAndAttributee<T>>> GetFunctionsWithAttribute<T>()
+			where T : BaseAttribute
 		{
 			if (typeof(T) == typeof(ObsoleteAttribute))
 			{
 				Loggy.LogError("hey no, no obsolete stuff");
```

Comment:
This error is unclear. You should report what exactly is having this issue.

Reply:
aaaa unrelated technically, shall I add it anyway